companion/src/eeprominterface.cpp

```diff
@@ -1843,3 +1844,16 @@ SimulatorInterface *GetCurrentFirmwareSimulator()
   else
     return NULL;
 }
+
+void FlightModeData::setDefaultLinkedFlightModes(const int phase)
```

Comment:
Perfectly understood, you did well. But I remember I did it in some other classes, that's why I ask if it is feasible here. Not completely sure it's used somewhere else to zeroe the structure